bst_plugin/process_nst_mbll.m

```diff
@@ -48,10 +48,14 @@
     sProcess.options.option_pvf.Type    = 'value';
     sProcess.options.option_pvf.Value   = {50, '', 0};
     
-    sProcess.options.option_do_plp_corr.Comment = 'Light path length correction';
+    sProcess.options.option_do_plp_corr.Comment = 'DPF correction';
     sProcess.options.option_do_plp_corr.Type    = 'checkbox';
     sProcess.options.option_do_plp_corr.Value   = 1;
     
+    sProcess.options.option_dpf_method.Comment = 'DPF method';
+    sProcess.options.option_dpf_method.Type    = 'combobox';
+    sProcess.options.option_dpf_method.Value   = {2, {'Scholkmann2013', 'Duncan interpolation'}};    % {Default index, {list of entries}}
```

Comment:
safer to define an enum-like structure to refer to dpf_method:

```
function dm = dpf_method_choices()
 dm.SCHOLKMANN2013 = 1;
 dm.DUNCAN = 2;
end
```

then use : `dpf_methods = dtp_methods_choices();
`